include/assimp/material.h

```diff
@@ -292,6 +292,11 @@ enum aiTextureType {
     aiTextureType_DIFFUSE_ROUGHNESS = 16,
     aiTextureType_AMBIENT_OCCLUSION = 17,
 
+    aiTextureType_MAYA_BASE = 22,
```

Comment:
Why not use the existing types?

Reply:
Existing types is not including support of Specular/Gloss textures.
<img width="219" alt="Screenshot 2023-05-22 at 14 02 01" src="https://github.com/assimp/assimp/assets/35233247/c3a8fb2c-fee9-4b9b-844f-d79f59a9ac30">